calibration/NUTNRM/CGINS-NUTNRM-00708__20150924.csv

```diff
@@ -1,8 +1,8 @@
-4,708,CC_lower_wavelength_limit_for_spectra_fit,217.0,217
-5,708,CC_upper_wavelength_limit_for_spectra_fit,240.0,240
-6,708,CC_wl,"[188.4, 189.21, 190.02, 190.82, 191.63, 192.43, 193.24, 194.05, 194.85, 195.66, 196.47, 197.28, 198.09, 198.89, 199.7, 200.51, 201.32, 202.13, 202.94, 203.75, 204.56, 205.37, 206.18, 206.99, 207.8, 208.61, 209.42, 210.23, 211.04, 211.86, 212.67, 213.48, 214.29, 215.1, 215.92, 216.73, 217.54, 218.36, 219.17, 219.98, 220.8, 221.61, 222.42, 223.24, 224.05, 224.87, 225.68, 226.5, 227.31, 228.13, 228.94, 229.76, 230.57, 231.39, 232.21, 233.02, 233.84, 234.65, 235.47, 236.29, 237.1, 237.92, 238.74, 239.56, 240.37, 241.19, 242.01, 242.83, 243.64, 244.46, 245.28, 246.1, 246.92, 247.73, 248.55, 249.37, 250.19, 251.01, 251.83, 252.65, 253.47, 254.29, 255.11, 255.92, 256.74, 257.56, 258.38, 259.2, 260.02, 260.84, 261.66, 262.48, 263.3, 264.12, 264.94, 265.76, 266.58, 267.41, 268.23, 269.05, 269.87, 270.69, 271.51, 272.33, 273.15, 273.97, 274.79, 275.61, 276.44, 277.26, 278.08, 278.9, 279.72, 280.54, 281.36, 282.18, 283.01, 283.83, 284.65, 285.47, 286.29, 287.11, 287.93, 288.76, 289.58, 290.4, 291.22, 292.04, 292.86, 293.69, 294.51, 295.33, 296.15, 296.97, 297.79, 298.62, 299.44, 300.26, 301.08, 301.9, 302.72, 303.54, 304.37, 305.19, 306.01, 306.83, 307.65, 308.47, 309.29, 310.12, 310.94, 311.76, 312.58, 313.4, 314.22, 315.04, 315.86, 316.68, 317.51, 318.33, 319.15, 319.97, 320.79, 321.61, 322.43, 323.25, 324.07, 324.89, 325.71, 326.53, 327.35, 328.17, 328.99, 329.81, 330.63, 331.45, 332.27, 333.09, 333.91, 334.73, 335.55, 336.37, 337.19, 338.0, 338.82, 339.64, 340.46, 341.28, 342.1, 342.92, 343.73, 344.55, 345.37, 346.19, 347.01, 347.82, 348.64, 349.46, 350.28, 351.09, 351.91, 352.73, 353.54, 354.36, 355.18, 355.99, 356.81, 357.62, 358.44, 359.26, 360.07, 360.89, 361.7, 362.52, 363.33, 364.15, 364.96, 365.77, 366.59, 367.4, 368.22, 369.03, 369.84, 370.66, 371.47, 372.28, 373.1, 373.91, 374.72, 375.53, 376.35, 377.16, 377.97, 378.78, 379.59, 380.4, 381.21, 382.02, 382.83, 383.64, 384.45, 385.26, 386.07, 386.88, 387.69, 388.5, 389.31, 390.12, 390.93, 391.73, 392.54, 393.35, 394.16, 394.96, 395.77, 396.58]",
+serial,name,value,notes
+708,CC_cal_temp,20.01, Source file: NUTNR-M_SUNA_SN_708_Calibration_Files_2015-09-24.zip > SNA0708B.cal
```

Comment:
Edit filename to: NUTNR-M_SUNA_SN_708_Calibration_2015-09-24.zip

Reply:
was only editing to remove the extra column, but ok - updated source file too